Dockerfile-alpine.template

```diff
@@ -58,11 +58,10 @@ RUN apk add --no-cache --virtual .build-deps-yarn curl gnupg tar \
   && curl -fSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz" \
   && curl -fSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz.asc" \
   && gpg --batch --verify yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
-  && mkdir -p /opt/yarn \
+  && mkdir /opt \
```

Comment:
I'd like to see this changed to `&& mkdir -p /opt \` and add it to all the templates and Dockerfiles. We like to keep the Dockerfiles and templates consistent across variants

Reply:
fixed